src/hxmath/geom/Ray2.hx

```diff
@@ -89,4 +89,4 @@ class Ray2
         
         return hitIndex;
     }
-}
```

Comment:
Minor: can you revert this block?

Reply:
It seems trailing newlines are so standard nowadays that the GitHub editor automatically adds them.